sql/item_mods.sql

```diff
@@ -59330,6 +59330,9 @@ INSERT INTO `item_mods` VALUES (25708,384,300); -- HASTE_GEAR: 300
 -- Track Shirt +1
 INSERT INTO `item_mods` VALUES (25714,64,1); -- COMBAT_SKILLUP_RATE: 1
 
+-- Track Pants +1
+INSERT INTO `item_mods` VALUES (27326,76,8);  -- MOVE_SPEED_GEAR_BONUS: 8
+
```

Comment:
While the change is good, the item is out of order.
25714
27326 <-- Yours
25718

Reply:
While you are here, add the DEF mod aswell
```sql
INSERT INTO `item_mods` VALUES (27326,1,2);    -- DEF: 2
INSERT INTO `item_mods` VALUES (27326,76,8);  -- MOVE_SPEED_GEAR_BONUS: 8
```
Item IDs go in numerical order. Then mods go in numerical order aswell